app/code/core/Mage/Directory/data/directory_setup/data-upgrade-1.6.0.3-1.6.0.4.php

```diff
@@ -0,0 +1,35 @@
+<?php
+/**
+ * OpenMage
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.txt.
+ * It is also available at https://opensource.org/license/osl-3-0-php
+ *
+ * @category   Mage
+ * @package    Mage_Directory
+ * @copyright  Copyright (c) 2023 The OpenMage Contributors (https://www.openmage.org)
+ * @license    https://opensource.org/licenses/osl-3.0.php  Open Software License (OSL 3.0)
+ */
+
+/** @var Mage_Core_Model_Resource_Setup $installer */
+$installer = $this;
+
+// The Netherlands Antilles (AN) was divided into Bonaire, Saint Eustatius and Saba (BQ, BES, 535),
+// Curaçao (CW, CUW, 531) and Sint Maarten (Dutch part) (SX, SXM, 534).
+//
+// See: https://www.iso.org/obp/ui/#iso:code:3166:AN
+$installer->getConnection()->delete(
+    $installer->getTable('directory/country'),
+    ['country_id = ?' => 'AN']
+);
+
+$data = [
+    ['BQ', 'BQ', 'BES'],
+    ['CW', 'CW', 'CUW'],
+    ['SS', 'SS', 'SSD'],
+    ['SX', 'SX', 'SXM'],
```

Comment:
We should also add North-Macedonia as said in [this discussion](https://github.com/OpenMage/magento-lts/issues/860). I think we should also add other countries added or removed after the first version of Magento 1 (not easy to do).

Reply:
North Macedonia has the country code MK which already exists.

I did a comparison between the countries in the database currently and the ones from the Wikipedia link and these are all the differences I found.